### ISSUE TITLE
v0.10.5 - ValidationError on copy annotation, upgrading from v0.10.2

### ISSUE THREAD

**Issue by EMCP** [jsbroks/coco-annotator#206]
upgraded recently.. and retried the annotation copy mechanism.. clicked the copy button to get the previous images annotations but got an error

```
annotator_webclient | [2019-05-06 20:18:12 +0000] [10] [DEBUG] POST /api/annotator/data
annotator_webclient | [2019-05-06 20:18:12 +0000] [10] [DEBUG] POST /api/image/copy/3332/3333/annotations
annotator_webclient | [2019-05-06 20:18:12 +0000] [10] [ERROR] Exception on /api/image/copy/3332/3333/annotations [POST]
annotator_webclient | Traceback (most recent call last):
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
annotator_webclient |     rv = self.dispatch_request()
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
annotator_webclient |     return self.view_functions[rule.endpoint](**req.view_args)
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/flask_restplus/api.py", line 325, in wrapper
annotator_webclient |     resp = resource(*args, **kwargs)
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/flask/views.py", line 88, in view
annotator_webclient |     return self.dispatch_request(*args, **kwargs)
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/flask_restplus/resource.py", line 44, in dispatch_request
annotator_webclient |     resp = meth(*args, **kwargs)
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
annotator_webclient |     return func(*args, **kwargs)
annotator_webclient |   File "/workspace/webserver/api/images.py", line 190, in post
annotator_webclient |     return {'annotations_created': image_to.copy_annotations(query)}
annotator_webclient |   File "/workspace/database/images.py", line 154, in copy_annotations
annotator_webclient |     clone.save(copy=True)
annotator_webclient |   File "/workspace/database/annotations.py", line 76, in save
annotator_webclient |     return super(AnnotationModel, self).save(*args, **kwargs)
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/mongoengine/document.py", line 369, in save
annotator_webclient |     self.validate(clean=clean)
annotator_webclient |   File "/usr/local/lib/python3.6/site-packages/mongoengine/base/document.py", line 392, in validate
annotator_webclient |     raise ValidationError(message, errors=errors)
annotator_webclient | mongoengine.errors.ValidationError: ValidationError (AnnotationModel:957) (Invalid embedded document instance provided to an EmbeddedDocumentField: ['events'])


```

**Comment by yinigao** [comment]
Note to someone who still have the problems:
Already fixed the issue, but you have to change docker-compose.yml to pull webserver-latest and worker-latest instead of stable version of the images.

**Comment by EMCP** [comment]
When is the stable version going to get this update I wonder

**Comment by jsbroks** [comment]
Should already be released in 10.6

**Comment by EMCP** [comment]
ah, missed that I still have `v10.5` from a while back on my local cache

a `docker-compose pull` and I am now on `v10.6`

thank you!
